source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/display.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { ProjectResource, attributesFor } from '@data';
+import { i18nProps } from '@lib/i18n';
+
```

Comment:
spacing?

Reply:
I need to add `prettier`.
We should have humans linting :)